src/solana_runtime/send_and_get_status_memo.rs

```diff
@@ -28,33 +30,24 @@ fn create_memo_tx(msg: &[u8], payer: &Keypair, blockhash: Hash) -> Transaction {
 
 #[async_trait]
 impl TestingTask for SendAndConfrimTesting {
-    async fn test(
-        &self,
-        args: crate::cli::Args,
-        config: crate::config::Config,
-    ) -> anyhow::Result<Stats> {
+    async fn test(&self, args: &Args, config: &Config) -> anyhow::Result<Stats> {
         let instant = SendMemoTransactionsBench {
             block_hash: self.block_hash.clone(),
-            payers: config
-                .users
-                .iter()
-                .map(|x| Arc::new(x.get_keypair()))
-                .collect(),
+            payers: Arc::new(config.users.iter().map(|x| x.get_keypair()).collect()),
         };
         let metric = Bencher::bench::<SendMemoTransactionsBench>(instant, args).await?;
-        log::info!("{} {}", self.get_name(), serde_json::to_string(&metric)?);
         Ok(metric)
     }
 
-    fn get_name(&self) -> String {
```

Comment:
Why convert to `static str`, This is not called million times per second anyways this will not create a bottleneck for performance. Moreover String could be used more freely. I prefer it to be String and not `static string.